kodilla-advanced-tests/src/test/java/com/kodilla/parametrized_tests/homework/GamblingMachineTestSuite.java

```diff
@@ -0,0 +1,80 @@
+package com.kodilla.parametrized_tests.homework;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvFileSource;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class GamblingMachineTestSuite {
+
+    @ParameterizedTest
+    @CsvFileSource(resources = "/GoodNumbers.csv")
+    public void shouldCheckWinsWork(String numbers) throws InvalidNumbersException {
+        String[] parsedNumbers = numbers.split("4,9,43,25,37,38");
+        Set numbersInt = new HashSet<>();
+        for (String number : parsedNumbers) {
+            numbersInt.add(Integer.valueOf(number));
+
+            GamblingMachine gamblingMachine = new GamblingMachine();
+            Set<Integer> set = new HashSet<>();
+            set.add(4);
```

Comment:
co to jest? Testujemy tylko fakt, ze nie polecial wyjatek.

Reply:
to jakiś konflikt ze starym plikiem